Bootstrap.php

```diff
@@ -865,7 +866,12 @@ protected function addCustomerTagging(Enlight_View_Default $view)
         if ($customerDataAllowed && $customer instanceof CustomerModel) {
             $nostoCustomer = new NostoCustomerModel();
             $nostoCustomer->loadData($customer);
-            $view->assign('nostoCustomer', $nostoCustomer);
+            // Add Customer HTML Tagging to Page
+            $view->extendsBlock(
```

Comment:
How about introducing a method `appendTagging(Enlight_View_Default $view, AbstarctObject $object)` and here we would only call `$this->appendTagging($view, $nostoCustomer)`